apply_extra.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# global variables
+FLATPAK_ID='__FLATPAK_ID__'
+
+# unpack debian package
+bsdtar -Oxf "Breitbandmessung-linux.deb" 'data.tar.*' |
+    bsdtar -xf -
+
+# /extra/usr/share => /extra/export/share
+mkdir export
+mv usr/share export/share
+rm -rf usr
+
+# .desktop file
+mv export/share/applications/breitbandmessung.desktop "export/share/applications/$FLATPAK_ID.desktop"
+sed -Ei 's|^Exec=.*|Exec=breitbandmessung|' "export/share/applications/$FLATPAK_ID.desktop"
+sed -Ei "s|^Icon=.*|Icon=$FLATPAK_ID|" "export/share/applications/$FLATPAK_ID.desktop"
```

Comment:
It is preferred when desktop and icon files are "local", not from extra-data.

Reply:
I think this would violate the license.

Reply:
It is nominative fair use. This app is hardly the first proprietary app on Flathub.

Reply:
done